server/roomlogs.ts

```diff
@@ -223,13 +222,13 @@ export class Roomlog {
 		message = message.replace(/<img[^>]* src="data:image\/png;base64,[^">]+"[^>]*>/g, '');
 		void this.roomlogStream.write(timestamp + message + '\n');
 	}
-	modlog(message: string, overrideID?: string) {
-		void Rooms.Modlog.write(this.roomid, message, overrideID);
+	modlog(entry: ModlogEntry) {
+		void Rooms.Modlog.write(this.roomid, entry);
 	}
 	async rename(newID: RoomID): Promise<true> {
 		const roomlogPath = `logs/chat`;
 		const roomlogStreamExisted = this.roomlogStream !== null;
-		await this.destroy(false); // don't destroy modlog, since it's renamed later
+		await this.destroy(); // don't destroy modlog, since it's renamed later
```

Comment:
What does this comment mean? Is it still relevant?

Reply:
It's not